src/iosrtcPlugin.swift

```diff
@@ -844,14 +804,36 @@ class iosrtcPlugin : CDVPlugin {
 			PluginEnumerateDevices.call(
 				{ (data: NSDictionary) -> Void in
 					self.emit(command.callbackId,
-						result: CDVPluginResult(status: CDVCommandStatus_OK, messageAs: data as! [AnyHashable: Any])
+                              result: CDVPluginResult(status: CDVCommandStatus_OK, messageAs: data as! [AnyHashable: Any])
 					)
 				}
 			)
 		}
 	}
 
 
+	func selectAudioOutputEarpiece(_ command: CDVInvokedUrlCommand) {
```

Comment:
What is this for? I don't see this `selectAudioOutputEarpiece` function is used anywhere. Why adding this unused function into a PR that is just supposed to update Swift code?